qa/run-standalone.sh

```diff
@@ -55,6 +55,9 @@ PATH=$(pwd)/bin:$PATH
 # user's PATH does not get these directories by default (e.g., tumbleweed)
 PATH=$PATH:/sbin:/usr/sbin
 
+export LD_LIBRARY_PATH="$(pwd)/lib"
+sudo ldconfig
```

Comment:
no need to run `ldconfig`, as we are not changing `/etc/ld.so.conf` here.

Reply:
I'll fix this.